### DISCUSSION TITLE
Setting Language Parsers for customer Source Member Types

### DISCUSSION THREAD

**Discussion by gcraig88keys** [codefori/vscode-ibmi#2127] [Q&A] [answered]
We use a member type, RPGLENC, for source members that contain our type definitions, prototypes for service programs, common named constants, etc.  In RDi, I can associate that member type with the RPGLE language parser.  Is there a way to do this in Code for IBM i?

**Comment by sebjulliand** [comment, marked as answer]
That relies on VSCode file associations.

1. Open one of your RPGLENC file.
2. At the bottom right, click on the language, whatever it may be
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/688f1d84-c7e1-4aba-8dde-77eaf315e594)
3.Select `Configure file association...`
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/d7f34210-a90f-4788-a929-9a5aae345bcd)
4. Look for `RPGLE` in the list and select it
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/1a5a7164-5b7f-498f-b94f-337290c3fa89)

**Reply by gcraig88keys** [reply]
Thanks!

**Reply by sebjulliand** [reply]
Anytime!
